draft-packages/form/KaizenDraft/Form/Primitives/InputSearch/InputSearch.tsx

```diff
@@ -32,7 +32,12 @@ export const InputSearch: React.VFC<InputSearchProps> = (
     secondary = false,
     ...restProps
   } = props
+  const inputRef = useRef<HTMLInputElement>(null)
 
+  const handleOnClear = (): void => {
+    inputRef.current?.focus()
+    onClear && onClear()
```

Comment:
```suggestion
    onClear?.()
```

Reply:
oo awesome nice to know - what do you call this?

Reply:
It's another way to use Optional Chaining:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining